labs/8.md

```diff
@@ -1,153 +1,39 @@
-## Deploying MongoDB
-There are many different types of federated objects but they are somewhat similar to those
-non-federated objects. For more information about federated objects see the following  [examples](https://github.com/kubernetes-sigs/kubefed/tree/master/example/sample1) and
-the [user guide](https://github.com/kubernetes-sigs/kubefed/blob/master/docs/userguide.md).
+# Application Portability
+The strength of KubeFed is the ability to mange application workloads and move applications between clusters.
 
-
-The first step is to clone the demo code to your local machine:
+## Moving the Application
```

Comment:
I'll add an example patching the placement so students can see the difference between modifying the replicas (deployment scaled to 0) and modifying the placement (deployment deleted)

Reply:
card created